.github/workflows/main.yml

```diff
@@ -41,6 +41,13 @@ jobs:
           echo 'gem: --no-document' | sudo tee /etc/gemrc
           cd _build
           bundle install
+      - name: Install Pygments
+        run: git clone -b 2.5.2 --depth 1 https://github.com/pygments/pygments.git
```

Comment:
Any particular reason against using Ubuntu's `python3-pygments` package?

Reply:
Ah ok, that explains it - yeah, Ubuntu's packages are often quite old... In that case it probably makes more sense to do a regular `python3 -m pip install --user pygments` to install the latest release version of Pygments, rather than cloning a specific version and installing it manually.